db/seeds.rb

```diff
@@ -125,9 +111,22 @@ def create_project_medias_with_channel(user, project, team, data)
 end
 
 def create_tipline_user_and_data(project_media, team)
+  tipline_message_data = {
+    link: 'https://www.nytimes.com/interactive/2023/09/28/world/europe/russia-ukraine-war-map-front-line.html',
+    audio: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/j5x5g1L28Y7fDrzU9DSwuwUR/wnHkwjykxOqU3SMWpEpuVzSa.oga',
+    video: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/HEFL3cuzvNnCDXZd0ElYziN2/AOVFpYOfMm_ssRUizUQhJHDD.mp4',
+    image: 'https://media.smooch.io/apps/5c193e2380a9c90022e40b86/conversations/d445678ac87d2e9f44485e40/sUn5Sq9xRiacbV0Af2qmjl5G/bOoeoeV9zNA51ecial0eWDG6.jpeg',
```

Comment:
Please replace these by fake URLs (we're moving out of Smooch anyway). The important bit here is just the file extensions, but the URLs don't need to be real.